web/manifest/manifest.go

```diff
@@ -0,0 +1,200 @@
+package manifest
+
+import (
+	"context"
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/internal/env"
+)
+
+// TODO(meling) Should reuse those in env package.
+const (
+	defaultPath  = "internal/config/github/quickfeed.pem"
+	appID        = "QUICKFEED_APP_ID"
+	appKey       = "QUICKFEED_APP_KEY"
+	clientID     = "QUICKFEED_CLIENT_ID"
+	clientSecret = "QUICKFEED_CLIENT_SECRET"
+)
+
+type Manifest struct {
+	http.Server
+	done chan bool
+}
+
+func NewManifest(addr string) *Manifest {
+	router := http.NewServeMux()
+	m := &Manifest{
+		done: make(chan bool),
+	}
+	router.Handle("/manifest/callback", m.Conversion())
+	router.Handle("/manifest", m.CreateApp())
+	m.Server = http.Server{
+		Addr:    addr,
+		Handler: router,
+	}
+	return m
+}
+
+func StartFlow(addr string) error {
```

Comment:
Could we use a less generic name? Something like
- `StartAppCreationFlow`
- `StartAppCreation`
- `StartAppFlow`